framework/include/controls/RealFunctionControl.h

```diff
@@ -62,7 +62,7 @@ class RealFunctionControl : public Control
   Function & _function;
 
   /// Vector of parameters to change
-  std::vector<Real *> _parameters;
+  ControllableParameter<Real> _parameters;
```

Comment:
There is nothing to reference, the object that is returned holds onto pointers. To make this a reference, it would need to be stored somewhere else for the sole purpose of providing a reference.